galaxy_ng/app/api/v3/urls.py

```diff
@@ -16,6 +16,18 @@
     path("auth/token/", views.TokenView.as_view(), name="auth-token"),
 ]
 
+sync_urls = [
```

Comment:
please add a note saying that these are included in the base router so that they only appear under `content/<distro>/v3` and not `v3/`